app/code/Magento/Backend/Block/System/Store/Grid/Render/Group.php

```diff
@@ -27,6 +27,7 @@ public function render(\Magento\Framework\DataObject $row)
         $this->getUrl('adminhtml/*/editGroup', ['group_id' => $row->getGroupId()]) .
         '">' .
         $this->escapeHtml($row->getData($this->getColumn()->getIndex())) .
-        '</a>';
+        '</a><br />'
+        . '(Code: ' . $row->getGroupCode() . ')';
```

Comment:
@aschrammel Please can you make "Code" translatable in all 3 places (like "Edit Store" here: [Group.php](https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Backend/Block/System/Store/Grid/Render/Group.php#L23-L25) and [en_US.csv](https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Backend/i18n/en_US.csv)?

Then we should be good to go. Thanks!